website/src/app/kb/deploy/gateways/readme.mdx

```diff
@@ -208,6 +208,18 @@ By default, Gateways will run a https://sentry.io crash-reporting agent. If
 you'd like to opt-out of this, set the environment variable
 `FIREZONE_NO_TELEMETRY=true`.
 
+## Environment variable reference
+
+| Variable Name           | Default Value | Description                                                                                                                                                                                                                                                    |
+| ----------------------- | ------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `FIREZONE_TOKEN`        |               | Token generated by the portal to authorize the Gateway's connection. See [Deploy multiple Gateways](#deploy-multiple-gateways) for where to find this. Can also be passed using a [systemd credential](https://systemd.io/CREDENTIALS) named `FIREZONE_TOKEN`. |
+| `FIREZONE_ID`           |               | Unique identifier for this Gateway. We recommend using a 64-character hex string, e.g. `$(head -c 32 /dev/urandom \| sha256sum \| cut -d' ' -f1)`.                                                                                                             |
+| `FIREZONE_NAME`         |               | Friendly name for this Gateway to display in the admin portal.                                                                                                                                                                                                 |
+| `FIREZONE_NO_TELEMETRY` | `false`       | Set to `true` to disable the Sentry.io crash-reporting agent.                                                                                                                                                                                                  |
+| `FIREZONE_LOG_FORMAT`   | `human`       | Log output format. Set to `json` for JSON-formatted logs, or `human` for human-readable logs.                                                                                                                                                                  |
+| `FIREZONE_NO_INC_BUF`   | `false`       | Set to `true` to prevent the Gateway from attempting to increase the system's `net.core.wmem_max` and `net.core.rmem_max` kernel parameters. See [Performance tuning](#performance-tuning) for details.                                                        |
+| `RUST_LOG`              | `info`        | Log level for the Gateway. Common values: `error`, `warn`, `info`, `debug`, `trace`. For flow logs, set to `flow_logs=trace`. Read more [here](https://docs.rs/env_logger/latest/env_logger/#enabling-logging).                                                |
```

Comment:
We use tracing, not `env_logger` so this reference is not strictly correct, even though the syntax is very similar: https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.EnvFilter.html#directives

Reply:
updated